src/plot_cartogram.R

```diff
@@ -101,7 +104,7 @@ plot_national_area <- function(national_data, date_start, date_end, pal, color_b
                        )) +
     theme_flowfacet(base = 12, color_bknd, text_color) +
     theme(axis.text.y = 
-            element_text(size = 12, 
+            element_text(size = 12,
                          vjust = c(1, 0), 
                          hjust = 1),
           axis.title.x.bottom = element_text(size = 20,
```

Comment:
does it make sense to move these theme elements to `theme_flowfacet`?

Reply:
This messes with the `plot_state_cartogram` that also uses `theme_flowfacet` ![flow_cartogram](https://user-images.githubusercontent.com/69014273/209222000-4a3731b0-f0b1-4378-a94d-0541a612ae93.png)